dev/pipenv.ts

```diff
@@ -0,0 +1,184 @@
+import { packageList } from "./pip";
```

Comment:
There are some options missing in the spec, if you do not intend to add those in this PR I would add this message for future contributors
```suggestion
// https://pipenv.pypa.io/en/stable/cli/
// TODO: complete spec
import { packageList } from "./pip";
```